src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -367,7 +367,7 @@ private void AddClaimsFromJToken(List<Claim> claims, string claimType, JToken jt
         {
             if (jtoken.Type is JTokenType.Object)
             {
-                claims.Add(new Claim(claimType, jtoken.ToString(Newtonsoft.Json.Formatting.None), JsonClaimValueTypes.Json, issuer, issuer));
+                claims.Add(new Claim(claimType, jtoken.ToString(Microsoft.IdentityModel.Json.Formatting.None), JsonClaimValueTypes.Json, issuer, issuer));
```

Comment:
>Microsoft.IdentityModel. [](start = 64, length = 24)

nit: Do you need the MS.IdentityModel stuff here?

Reply:
These are remnants of replacing Newtonsoft.Json with Microsoft.IdentityModel.Json. I'll make the change as I don't really need FQN here.